gctscript/modules/gct/exchange.go

```diff
@@ -213,15 +213,23 @@ func ExchangePairs(args ...objects.Object) (objects.Object, error) {
 
 // ExchangeAccountInfo returns account information for requested exchange
 func ExchangeAccountInfo(args ...objects.Object) (objects.Object, error) {
-	if len(args) != 1 {
+	if len(args) != 2 {
```

Comment:
Please update the example script `account_info.gct` to have two args and double check no other example script is missing accountinfo args

Reply:
All the other scripts seem to have assetType in them already 🤞